Formula/nef.rb

```diff
@@ -0,0 +1,20 @@
+class Nef < Formula
+  desc "💊 steroids for Xcode Playgrounds"
+  homepage "https://nef.bow-swift.io"
+  url "https://github.com/bow-swift/nef/archive/0.6.0.tar.gz"
+  sha256 "a67bb4201739898832ec52e92d838ccfcae25a490df433cd7cf44f1fc6e0a786"
+
+  depends_on :xcode => "11.0"
+
+  def install
+    system "make", "install", "prefix=#{prefix}", "version=#{version}"
+  end
+
+  test do
+    rm_rf "/tmp/nef-#{version}-tests"
```

Comment:
Please just write to the local directory or a folder within it. If it needs the full name you can use `#{testpath}`. Tests are run in a directory that gets removed after.

Reply:
Done! I did not know about this alias, thanks ^^ it was helpful